SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -435,10 +446,62 @@ inline Worker_ComponentId RPCTypeToWorkerComponentIdLegacy(ERPCType RPCType)
 	}
 }
 
-inline Worker_ComponentId GetClientAuthorityComponent(bool bUsingRingBuffers)
-{
-	return bUsingRingBuffers ? CLIENT_ENDPOINT_COMPONENT_ID : CLIENT_RPC_ENDPOINT_COMPONENT_ID_LEGACY;
-}
+const TArray<FString> ServerAuthorityWellKnownSchemaImports = {
+	"improbable/standard_library.schema",
+	"unreal/gdk/authority_intent.schema",
+	"unreal/gdk/debug_component.schema",
+	"unreal/gdk/debug_metrics.schema",
+	"unreal/gdk/net_owning_client_worker.schema",
+	"unreal/gdk/not_streamed.schema",
+	"unreal/gdk/query_tags.schema",
+	"unreal/gdk/relevant.schema",
+	"unreal/gdk/rpc_components.schema",
+	"unreal/gdk/spatial_debugging.schema",
+	"unreal/gdk/spawndata.schema",
+	"unreal/gdk/tombstone.schema",
+	"unreal/gdk/unreal_metadata.schema",
+	"unreal/generated/rpc_endpoints.schema",
+	"unreal/generated/NetCullDistance/ncdcomponents.schema",
+};
+
+const TMap<Worker_ComponentId, FString> ServerAuthorityWellKnownComponents = {
```

Comment:
maybe not for this PR, but could we read these arrays/maps out of the schema bundle rather than hardcode them?

Reply:
I think we also had this discussion offline, so my bad- glad it's tracked

Reply:
Will add a comment